src/schema/v2/fairs.ts

```diff
@@ -83,3 +87,72 @@ const Fairs: GraphQLFieldConfig<void, ResolverContext> = {
 }
 
 export default Fairs
+
+export const fairsConnection = {
+  type: fairConnection.connectionType,
+  args: pageable({
+    fairOrganizerID: { type: GraphQLString },
+    hasFullFeature: { type: GraphQLBoolean },
+    hasHomepageSection: { type: GraphQLBoolean },
+    hasListing: { type: GraphQLBoolean },
+    ids: {
+      type: new GraphQLList(GraphQLString),
+      description:
+        "Only return fairs matching specified IDs. Accepts list of IDs.",
+    },
+    near: { type: Near },
+    sort: FairSorts,
+    status: EventStatus,
+  }),
+  description: "A list of fairs",
+  resolve: async (
+    _root,
+    args: {
+      fairOrganizerID?: string
+      hasFullFeature?: boolean
+      hasHomepageSection?: boolean
+      hasListing?: boolean
+      ids?: string[]
+      near?: TNear
+      sort?: TFairSorts
+      status?: TEventStatus
+    } & CursorPageable,
+    { fairsLoader }
+  ) => {
+    const { size, offset, page } = convertConnectionArgsToGravityArgs(args)
+
+    const { body, headers } = await fairsLoader({
+      fair_organizer_id: args.fairOrganizerID,
+      has_full_feature: args.hasFullFeature,
+      has_homepage_section: args.hasHomepageSection,
+      has_listing: args.hasListing,
+      id: args.ids,
+      page,
+      size,
+      sort: args.sort,
```

Comment:
y not `sort, status` (like the above two lines)

Reply:
`size` and `page` are coming from the connection conversion above, rather than the args. I could just destructure out all the args and rename them, true. But it probably adds to the line count anyway.